issuetracker/internal/redmine/redmine.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // New creates a new redmine issuetracker instance
-func New(origin string, authCfg *config.Auth) (*IssueTracker, error) {
+func New(authCfg *config.Auth, origin string) (*IssueTracker, error) {
```

Comment:
Why swap the func arguments?

Reply:
On the first commit `06d7321` the way IssueTracker.New worked needed all 3 arguments to be passed(issuetracker, config, origin) so i used them in the same order as factory.go. After moving the responsibility of the validation to the validation package i removed the IssueTracker but did not revert the order(which is better as origin,cfg to be in par with the struct).

I reverted this again, sorry for causing this confusion :)